template.yaml

```diff
@@ -1918,17 +1918,32 @@ Resources:
       Events:
         FanoutSource:
           Type: DynamoDB
+#==============================PROPERTIES WE WILL TEST=======================================================================
           Properties:
             Stream: !GetAtt NvaResourcesTable.StreamArn
             StartingPosition: TRIM_HORIZON
-            BatchSize: 1
+            BatchSize: 100
+            MaximumBatchingWindowInSeconds: 2
```

Comment:
Suggest setting 1 to keep round trip time as low as possible? like when we do publish on a resource, its important to keep latency low.

Maybe even skip MaximumBatchingWindowInSeconds (keep it at 0) and set BatchSize to 10